guide/06-imagery-and-raster-analysis/lidar_basics_part1.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<p style="user-select: auto;">"The list below includes common examples of...." -&gt; Below is a list of different things you can do with LAS Datasets</p><p style="user-select: auto;">"Quickly display LiDAR data as point clouds or <strong>as</strong> a triangulated surface in 2D and 3D." (added the word 'as')</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/610/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>